attributes/defaults.rb

```diff
@@ -54,6 +66,26 @@
   default['mysqld']['my.cnf']['mysqld_safe']['skip_log_error'] = true
 end
 
+# Password columns for user passwords
+case node['mysqld']['db_install']
+when 'mariadb'
+  default['mysqld']['pwd_col'] = 'Password'
+when 'percona'
+  if node['mysqld']['repository']['percona']['version'] == '5.6'
+    default['mysqld']['pwd_col'] = 'Password'
+  else
+    default['mysqld']['pwd_col'] = 'authentication_string'
+  end
+when 'mysql'
+  if node['platform'] == 'debian' && node['platform_version'].to_f < 9.0
```

Comment:
It seems like we should add an attribute `node['mysqld']['mysql']['version']` analogue to the percona configuration and check for the version instead of `node['platform_version']`. The version attribute can then be defaulted for a specific platform.